batchflow/models/metrics/regression.py

```diff
@@ -0,0 +1,153 @@
+""" Contains regression metrics """
+import numpy as np
+
+from . import Metrics
+
+METRICS_ALIASES = {'mae': 'mean_absolute_error',
+                   'mse': 'mean_squared_error',
+                   'rmse': 'root_mean_squared_error',
+                   'r2': 'r2_score',
+                   'acc': 'accuracy'}
+
+class RegressionMetrics(Metrics):
+    """ Metrics to assess regression models.
+
+    Available metrics: mae, mse, rmse, r2_score, explained_variance_ratio,
```

Comment:
Make it a proper [`reST` list](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.htmlrl)